troposphere/s3.yaml

```diff
@@ -0,0 +1,40 @@
+# AWS::Batch overrides
+header: |
+    try:
+        from awacs.aws import Policy
+
+        policytypes = (dict, Policy)
+    except ImportError:
+        policytypes = dict,
+
+    Private = "Private"
+    PublicRead = "PublicRead"
+    PublicReadWrite = "PublicReadWrite"
+    AuthenticatedRead = "AuthenticatedRead"
+    BucketOwnerRead = "BucketOwnerRead"
+    BucketOwnerFullControl = "BucketOwnerFullControl"
+    LogDeliveryWrite = "LogDeliveryWrite"
+classes:
+  CorsRules:
+    MaxAge:
```

Comment:
That likely is a good change. One thing I might put in there is something to allow renaming the class which we've had to do on occasion. Also, you may have noticed there are breaking changes due to renamed resource/properties which could be mapped for backward compatibility via either a rename or alias.

Reply:
@phobologic also take a look at #1301.

Reply:
Ugh...should have thought about adding this to the above comment...sorry for the spam.

Look at both this and the other WIP with respect to the impact on [stacker](https://github.com/cloudtools/stacker). Be good to understand those needs in case we change Python versions and/or how properties are used.